src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,125 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Climber extends SubsystemBase {
+    private final TalonFX master = new TalonFX(Ports.Climber.MASTER);
+    private final TalonFX slave = new TalonFX(Ports.Climber.SLAVE);
+    private final UnitModel unitModel = new UnitModel(Constants.Climber.TICKS_PER_METER);
+    private final Solenoid stopper = new Solenoid(Ports.Climber.STOPPER);
+    private final Solenoid gearboxPiston = new Solenoid(Ports.Climber.GEARBOX_PISTON);
+
+    public Climber() {

```

Comment:
Config selected sensor

Reply:
This are falcons they have an internal sensor

Reply:
You are right i thought they were SRX